### DISCUSSION TITLE
Welcome to secret-santa Discussions!

### DISCUSSION THREAD

**Discussion by mdwiltfong** [mdwiltfong/secret-santa#9] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ivavay** [comment]
Hi everyone! Excited to be working on this project with y'all. I've been dabbling in front-end dev for a couple of years now but have never worked on a group project so I'm excited to dive into this and learn some new things.
